cirq-core/cirq/work/sampler.py

```diff
@@ -263,6 +275,34 @@ def run_batch(
             ValueError: If length of `programs` is not equal to the length
                 of `params_list` or the length of `repetitions`.
         """
+        params_list, repetitions = self._normalize_batch_args(programs, params_list, repetitions)
+        return [
+            self.run_sweep(circuit, params=params, repetitions=repetitions)
+            for circuit, params, repetitions in zip(programs, params_list, repetitions)
+        ]
+
+    async def run_batch_async(
+        self,
+        programs: Sequence['cirq.AbstractCircuit'],
+        params_list: Optional[Sequence['cirq.Sweepable']] = None,
+        repetitions: Union[int, Sequence[int]] = 1,
+    ) -> Sequence[Sequence['cirq.Result']]:
+        """Runs the supplied circuits.
+
+        This is an asynchronous version of `run_batch`.
```

Comment:
Document args / return value.

Reply:
My idea here was just to refer to `run_batch` rather than repeat the same docs here. I've added a note to make this more explicit.

Reply:
Seems OK since these are on the same page in the docs site. Would be nice if we could link, but I'm not 100% certain whether `cirq.Sampler.run_batch` would auto-link properly with the tfdocs tools.